### PR TITLE
refactor(#606): organize controllers into folders

### LINKED ISSUE #606: Refactoring/Code Quality: Organize controllers into namespaces
### What needs to be done
We have a lot of controllers: 

TODO: 
- Organize them into namespaces (Folders) for better maintainability. 
- Perform any necessary refactoring. Ensure all tests run as expected. 

Namespace suggestions:
- Admin (required, should contain all admin controllers, create child namespaces if needed)
- Bookings
- User Profile
- System 

### Out of scope
any new functionality or changes not explicitly needed for the above

### PR DESCRIPTION
**Pull request by EvanDime**
##### _List all issues this PR resolves_
- closes #606 

##### _Complete these items before requesting reviews_
- [x] Static Analysis Rules Validated
- [x] All commits follow naming conventions

### CODE REVIEW

**Comment by EvanDime** [comment]
Made the sorting though they may not be in the perfect folders.

File: app/Http/Controllers/UserProfile/UserController.php
Comment (alexstojda) [inline app/Http/Controllers/UserProfile/UserController.php:3]: This should go to admin namespace

File: app/Http/Controllers/UserProfile/SocialLoginController.php
Comment (alexstojda) [inline app/Http/Controllers/UserProfile/SocialLoginController.php:3]: This one is more of a System controller

File: app/Http/Controllers/UserProfile/UserController.php
Comment (EvanDime) [reply app/Http/Controllers/UserProfile/UserController.php]: @alexstojda Agreed

File: app/Http/Controllers/UserProfile/SocialLoginController.php
Comment (EvanDime) [reply app/Http/Controllers/UserProfile/SocialLoginController.php]: @alexstojda Ditto

**Comment by alexstojda** [comment]
You have one smell to address https://sonarcloud.io/project/issues?id=CSU-Booking-Platform_application&pullRequest=619&resolved=false&types=CODE_SMELL

**Comment by EvanDime** [comment]
> You have one smell to address https://sonarcloud.io/project/issues?id=CSU-Booking-Platform_application&pullRequest=619&resolved=false&types=CODE_SMELL

@alexstojda I went over this last time but i'll do it again since i am force to re-touch this code. We cannot remove this unnused variable and we have no use for it. Laravel puts the attribute name as the first parameter, but we don't use it since we display errors using english instead of things like `time.0.start.time` .
There is no way to remove the code smell without just setting a variable and then unsetting it just to waist processing power.

**Comment by alexstojda** [comment]
> > You have one smell to address https://sonarcloud.io/project/issues?id=CSU-Booking-Platform_application&pullRequest=619&resolved=false&types=CODE_SMELL
> 
> @alexstojda I went over this last time but i'll do it again since i am force to re-touch this code. We cannot remove this unnused variable and we have no use for it. Laravel puts the attribute name as the first parameter, but we don't use it since we display errors using english instead of things like `time.0.start.time` .
> There is no way to remove the code smell without just setting a variable and then unsetting it just to waist processing power.

On the first line of that anonymous function, add `unset($attribute);` and it should supress the warning. The smell is for code quality. So when someone is reading the code and trying to understand what it does, they don't waste time trying to see what that parameter is for, when it is infact unused. 

So I suggest adding that code there with a comment explaining its un-used. Future maintainers may change that!

**Review by Alex-Hayeur — APPROVED** [review]
Very helpful for our own organization and being able to hand this off later. Good work.

File: app/Http/Controllers/Admin/BlackoutController.php
Comment (a-drew) [inline app/Http/Controllers/Admin/BlackoutController.php:147]: think this should've been after start_time

**Review by a-drew — APPROVED** [review]
other than that one typo, great job with cleaning up the controllers and making the project more maintainable

File: app/Http/Controllers/Admin/BlackoutController.php
Comment (a-drew) [reply app/Http/Controllers/Admin/BlackoutController.php:147]: oh rip @EvanDime the auto merge kicked in

Comment (EvanDime) [reply app/Http/Controllers/Admin/BlackoutController.php:147]: dude...you don't approve if you don't think it can merge xD
Either way is something we can fix later. this was just to get the smell away.  Also we can now just update teh constant and all the instances will work too

**Pull request merged**
